editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -60,8 +60,8 @@ describe('React Render Count Tests - ', () => {
     )
 
     const renderCountAfter = renderResult.getNumberOfRenders()
-    expect(renderCountAfter - renderCountBefore).toBeGreaterThan(280) // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toBeLessThan(290)
+    expect(renderCountAfter - renderCountBefore).toBeGreaterThan(320) // if this breaks, GREAT NEWS but update the test please :)
```

Comment:
Hmmmmmmmmm.....

Reply:
WELL, ACTUALLY this is purely because we are now running the monkey function for every test. but that is fine, as the real editor is also always running the monkey function